integration-tests/twitter/src/test/java/org/apache/camel/quarkus/component/twitter/CamelTwitterTest.java

```diff
@@ -80,8 +80,10 @@ public void e2e() {
         final long sinceId = Long.parseLong(messageId) - 1;
         /* Check that the message is seen in the timeline by the polling consumer */
         {
+            final int initialDelayMs = 60000;
             final int retries = 5;
             final int delayMs = 3000;
+            Thread.sleep(initialDelayMs);
```

Comment:
This will prolong the test execution by 1 min even if the message appears in the timeline much earlier. Wouldn't it be better to increase the number of retries? Or wouldn't some shorter initialDelayMs work too? like 10 seconds?

Reply:
@ppalaga Yes, in ideal world, it would be much better, but with such approach it leads to 
```
429:Returned in API v1.1 when a request cannot be served due to the application's rate limit having been exhausted for the resource. See Rate Limiting in API v1.1.(https://dev.twitter.com/docs/rate-limiting/1.1)
```